eval-server/python/src/bo_eval_server/openai_server.py

```
@@ -0,0 +1,644 @@
+                    if field in result and isinstance(result[field], str) and result[field].strip():
+                        return result[field]
+            
+            # If result is an object, try to extract meaningful content
+            return json.dumps(result, indent=2)
+        
+        return "Unable to extract response text from evaluation result"
+    
+    async def start(self):
+        """Start the HTTP server"""
+        if not self.eval_server.is_running():
+            raise RuntimeError("EvalServer must be started before starting OpenAI-compatible server")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)